src/modules/core/components/MembersList/Actions/MemberActionsPopover.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+import { Colony } from '~data/index';
+import DropdownMenu, {
+  DropdownMenuSection,
+  DropdownMenuItem,
+} from '~core/DropdownMenu';
+import Button from '~core/Button';
+import { useDialog } from '~core/Dialog';
+import { BanUserDialog } from '~core/Comment';
+import ExternalLink from '~core/ExternalLink';
+import ManageWhitelistDialog from '~dashboard/Dialogs/ManageWhitelistDialog';
+
+import styles from './MemberActionsPopover.css';
+
+const MSG = defineMessages({
+  addToAddressBook: {
+    id: 'core.MembersList.MemberActionsPopover.addToAddressBook',
+    defaultMessage: `Add to address book`,
+  },
+  banUser: {
+    id: 'core.MembersList.MemberActionsPopover.banUser',
+    defaultMessage: `{unban, select,
+      true {Unban}
+      other {Ban}
+    } user`,
+  },
+  viewOnBlockscout: {
+    id: 'core.MembersList.MemberActionsPopover.viewOnBlockscout',
+    defaultMessage: 'View on Blockscout',
+  },
+});
+
+interface Props {
+  closePopover: () => void;
+  colony: Colony;
+  userAddress: string;
+  isWhitelisted: boolean;
+  isBanned: boolean;
+  canAdministerComments?: boolean;
+}
+
+const displayName = 'core.MembersList.MemberActionsPopover';
+
+const MemberActionsPopover = ({
+  closePopover,
+  canAdministerComments,
+  colony,
+  userAddress,
+  isWhitelisted,
+  isBanned,
+}: Props) => {
+  const openBanUserDialog = useDialog(BanUserDialog);
+  const handleBanUser = useCallback(
+    () =>
+      openBanUserDialog({
+        colonyAddress: colony.colonyAddress,
+      }),
+    [openBanUserDialog, colony],
+  );
+  const openManageWhitelistDialog = useDialog(ManageWhitelistDialog);
+  const handleManageWhitelist = useCallback(
+    () =>
+      // @ts-ignore
```

Comment:
Why is this ignored ?

Reply:
Fixed. I was using the dialog as I saw it in other places, using it like this returned some confusing ts error, but I took a closer look and it seems like with a small adjustment this ignore won't be needed. So, I'll remove it here and others places where I saw it